<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWizardVdmSound</class>
 <widget class="QDialog" name="frmWizardVdmSound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asistente del VdmSound</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img16/cat/vdmsound.png</normaloff>:/img16/cat/vdmsound.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="imgLogoSvm">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/gr-lida.qrc">:/images/wizard_vdmsound.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_Datos_1">
        <property name="text">
         <string>Título:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="lb_Vdms_3">
        <property name="text">
         <string>Parametros:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnVdms_ExeJuego">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/img16/carpeta_1.png</normaloff>:/img16/carpeta_1.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLineEdit" name="txtVdms_path_exe">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="btnVdms_ExeJuego_clear">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="lb_Vdms_4">
        <property name="text">
         <string>Dirección del icono:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="txtVdms_icon">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnVdms_Icono">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/img16/carpeta_1.png</normaloff>:/img16/carpeta_1.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="txtVdms_params">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="vSpacer_0">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="btnVdms_Icono_clear">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="lb_Vdms_2">
        <property name="text">
         <string>Ejecutable (exe,com,bat) del juego:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="btnVdms_params_clear">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="txtDat_titulo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnDescargarInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Descargar la info del juego</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/img16/go-down.png</normaloff>:/img16/go-down.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img16/cancelar.png</normaloff>:/img16/cancelar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtDat_titulo</tabstop>
  <tabstop>btnVdms_ExeJuego</tabstop>
  <tabstop>txtVdms_path_exe</tabstop>
  <tabstop>btnVdms_ExeJuego_clear</tabstop>
  <tabstop>txtVdms_params</tabstop>
  <tabstop>btnVdms_params_clear</tabstop>
  <tabstop>btnVdms_Icono</tabstop>
  <tabstop>txtVdms_icon</tabstop>
  <tabstop>btnVdms_Icono_clear</tabstop>
  <tabstop>btnDescargarInfo</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
